### PR TITLE
test: Skip dmesg integration test for RHEL

### LINKED ISSUE #1450: skip dmesg integration test for RHEL
See https://github.com/kata-containers/tests/issues/1443

### PR DESCRIPTION
**Pull request by GabyCT**
While running this docker integration test on RHEL, we have
random failures in the jenkins ci
https://github.com/kata-containers/tests/issues/1443.

Fixes #1450

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: integration/docker/run_test.go [hunk 1]
```diff
@@ -23,6 +24,21 @@ func withWorkload(workload string, expectedExitCode int) TableEntry {
 	return Entry(fmt.Sprintf("with '%v' as workload", workload), workload, expectedExitCode)
 }
 
+func distroID() string {
+	pathFile := "/etc/os-release"
+	_, err := os.Stat(pathFile)
+	if _, err := os.Stat(pathFile); os.IsNotExist(err) {
```
Comment (devimc) [inline integration/docker/run_test.go]: double `Stat` ? :open_mouth:

File: integration/docker/run_test.go [hunk 2]
```diff
@@ -23,6 +24,21 @@ func withWorkload(workload string, expectedExitCode int) TableEntry {
...
+		pathFile = "/usr/lib/os-release"
+	}
+	cmd := exec.Command("sh", "-c", fmt.Sprintf("source %s; echo -n $ID", pathFile))
+	id, _:= cmd.CombinedOutput()
```
Comment (devimc) [inline integration/docker/run_test.go]: you're ignoring the error

File: integration/docker/run_test.go [hunk 3]
```diff
@@ -23,6 +24,21 @@ func withWorkload(workload string, expectedExitCode int) TableEntry {
...
+	if err != nil {
+		LogIfFail("couldn't find distro ID")
```
Comment (devimc) [inline integration/docker/run_test.go]: and the error?

**Comment by GabyCT** [comment]
@devimc changes applied

**Comment by devimc** [comment]
I was wondering if wouldn't be better to use the filtering file, https://github.com/kata-containers/tests/blob/master/Makefile#L16

**Pull request merged**